hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirErasureCodingOp.java

```diff
@@ -168,8 +168,8 @@ static FileStatus setErasureCodingPolicy(final FSNamesystem fsn,
       throw new FileNotFoundException("Path not found: " + srcIIP.getPath());
     }
     if (!inode.isDirectory()) {
-      throw new IOException("Attempt to set an erasure coding policy " +
-          "for a file " + src);
+      throw new IOException("Cannot set an erasure coding policy " +
+          "for a file " + src + ". Expected to be a directory");
```

Comment:
nit: "Expected to be a directory" -> "Erasure coding policy can only be set for directories."